app/models.py

```
@@ -127,37 +128,54 @@ class Certification(Document):
     certificate_url = StringField()
     created_at = DateTimeField(default=ist_now)
 
+from mongoengine import (
+    Document, StringField, DateTimeField, BooleanField,
+    ReferenceField, ListField, EmbeddedDocument, EmbeddedDocumentField, IntField, FloatField
```

Ruff (0.12.2)

Redefinition of unused `FloatField` from line 1

Remove definition: `FloatField`

(F811)